src/network/RTypeClient.cpp

```diff
@@ -24,7 +24,7 @@ void RTypeClient::engineInit()
     std::cout << "CompNb : " << (int)compNb << "." << std::endl;
     std::cout << "CompNameMaxSize : " << (int)compNameMaxSize << "." << std::endl;
 
-    for (uint16_t i; i < compNb; i++) {
+    for (uint16_t i = 0; i < compNb; i++) {
         std::vector<uint8_t> compName = blockingReceive();
         std::string strCompName = std::string(compName.begin() + 1, compName.end());
         // uint16_t CompId = (static_cast<uint16_t>(compName[0]) << 8) | static_cast<uint16_t>(compName[1]);
```

Comment:
comment left on purpose ?

Reply:
Yes It is for when we will switch to uint16 component type

Reply:
alright :+1: